### PR TITLE
Waiting Room to Car unit test coverage

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
WIP: Increased Unit Test Coverage

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by matthew2564 — APPROVED** [review]

File: src/app/pages/waiting-room-to-car/cat-adi-part2/__tests__/waiting-room-to-car.cat-adi-part2.page.spec.ts [hunk 1]
```diff
@@ -101,7 +98,7 @@ describe('WaitingRoomToCarCatADIPart2Page', () => {
         MockComponent(TransmissionComponent),
         MockComponent(TrainingRecordsCatAdiPart2Component),
         MockComponent(TrainerRegistrationNumberCatAdiPart2Component),
-        MockComponent(OrditTrainerCatAdiPart2Component),
+        MockComponent(WaitingRoomToCarCatADIPart2Page),
```
Comment (matthew2564) [inline src/app/pages/waiting-room-to-car/cat-adi-part2/__tests__/waiting-room-to-car.cat-adi-part2.page.spec.ts]: This needs to be reverted.

File: src/app/pages/waiting-room-to-car/cat-adi-part3/components/accompaniment-card/__tests__/accompaniment-card.spec.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import { AuthenticationProvider } from '@providers/authentication/authentication';
+import {
+  FormGroup, ReactiveFormsModule,
+} from '@angular/forms';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import {
+  AccompanimentCardADI3Component,
+} from '@pages/waiting-room-to-car/cat-adi-part3/components/accompaniment-card/accompaniment-card';
+
+describe('TrainingRecordsCatAdiPart2Component', () => {
```
Comment (matthew2564) [inline src/app/pages/waiting-room-to-car/cat-adi-part3/components/accompaniment-card/__tests__/accompaniment-card.spec.ts]: ```describe('AccompanimentCardADI3Component'```

File: src/app/pages/waiting-room-to-car/cat-adi-part3/components/waiting-room-to-car.cat-adi-part3.components.module.ts [hunk 3]
```diff
@@ -19,8 +15,8 @@ import { WaitingRoomToCarComponentsModule } from '../../components/waiting-room-
 
 @NgModule({
   declarations: [
-    DualControlsComponent,
-    PDILogbookComponent,
```
Comment (matthew2564) [inline src/app/pages/waiting-room-to-car/cat-adi-part3/components/waiting-room-to-car.cat-adi-part3.components.module.ts:23]: Why have these been removed?

Comment (matthew2564) [reply src/app/pages/waiting-room-to-car/cat-adi-part3/components/waiting-room-to-car.cat-adi-part3.components.module.ts:23]: Screenshot 2022-09-29 at 08 49 35

File: src/app/pages/waiting-room-to-car/cat-adi-part3/components/accompaniment-card/__tests__/accompaniment-card.spec.ts (hunk 2 above)
Comment (RLCorp) [reply src/app/pages/waiting-room-to-car/cat-adi-part3/components/accompaniment-card/__tests__/accompaniment-card.spec.ts]: addressed in newest commit.

File: src/app/pages/waiting-room-to-car/cat-adi-part3/components/waiting-room-to-car.cat-adi-part3.components.module.ts (hunk 3 above)
Comment (RLCorp) [reply src/app/pages/waiting-room-to-car/cat-adi-part3/components/waiting-room-to-car.cat-adi-part3.components.module.ts:23]: addressed in newest commit.

File: src/app/pages/waiting-room-to-car/cat-adi-part2/__tests__/waiting-room-to-car.cat-adi-part2.page.spec.ts (hunk 1 above)
Comment (RLCorp) [reply src/app/pages/waiting-room-to-car/cat-adi-part2/__tests__/waiting-room-to-car.cat-adi-part2.page.spec.ts]: addressed in newest commit.

**Review by gittins7 — APPROVED** [review]

**Pull request merged**